### PR TITLE
Updated knative.dev/serving to master

### PR DESCRIPTION
**Pull request by jmprusi**
/cc @pmorie

### CODE REVIEW

File: pkg/reconciler/ingress/controller.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package ingress
 
 import (
 	"context"
+	knative_reconciler "knative.dev/pkg/reconciler"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ package ingress
...
 	"strings"
 	"time"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"time"

	knative_reconciler "knative.dev/pkg/reconciler"
```

**Comment by pmorie** [comment]
@jmprusi looks like the test-infra dep was deleted: https://github.com/knative/net-kourier/pull/7/files#diff-cce40ed376e36675d8567fdf9515a26a ... which is causing the prow jobs to fail

**Comment by jmprusi** [comment]
let me check.. but about the prow jobs, we are working still on getting them fixed: https://github.com/knative/net-kourier/pull/6

**Comment by davidor** [comment]
@jmprusi please rebase this on top of master to pick the changes in #6 
After you do that, remember to change the serving version in `hack/update-deps.sh` as well and run the script.

**Comment by jmprusi** [comment]
> @jmprusi please rebase this on top of master to pick the changes in #6
> After you do that, remember to change the serving version in `hack/update-deps.sh` as well and run the script.

on it.. what a mess. :)

File: pkg/reconciler/ingress/controller.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ package ingress
 
 import (
 	"context"
+	reconciler_knative "knative.dev/pkg/reconciler"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go [hunk 4]
```diff
@@ -18,6 +18,7 @@ package ingress
...
 	"strings"
 	"time"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	reconciler_knative "knative.dev/pkg/reconciler"
```

File: pkg/reconciler/ingress/controller.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	reconciler_knative "knative.dev/pkg/reconciler"
```

File: pkg/reconciler/ingress/controller.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	reconciler_knative "knative.dev/pkg/reconciler"
```

File: pkg/reconciler/ingress/controller.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	reconciler_knative "knative.dev/pkg/reconciler"
```

**Comment by mattmoor** [comment]
Just a heads up that I have: https://github.com/knative/net-kourier/pull/10 which will conflict with this.  I think this change amounts to changing [this](https://github.com/knative/net-kourier/pull/10/files#diff-58d4b0ee04af7710eefa27cf91255798R32) line to `release-0.13`.

This is a pre-req to automation that'll open a PR daily to try to pull things forward and that PR should let us know within 24 hours whether something broke this repo upstream.

**Comment by jmprusi** [comment]
> Just a heads up that I have: #10 which will conflict with this. I think this change amounts to changing [this](https://github.com/knative/net-kourier/pull/10/files#diff-58d4b0ee04af7710eefa27cf91255798R32) line to `release-0.13`.
> 
> This is a pre-req to automation that'll open a PR daily to try to pull things forward and that PR should let us know within 24 hours whether something broke this repo upstream.

I will close this PR in favor of #10, we had to some ugly hacks (wget!). Also, test will not go green in your PR without https://github.com/knative/serving/pull/7470

**Comment by davidor** [comment]
@jmprusi @mattmoor I think we'll need this commit https://github.com/knative/net-kourier/pull/7/commits/f1104149acf9b35a63366e4f7b392ea85c81940d in the new PR.

**Comment by jmprusi** [comment]
> @jmprusi @mattmoor I think we'll need this commit [f110414](https://github.com/knative/net-kourier/commit/f1104149acf9b35a63366e4f7b392ea85c81940d) in the new PR.

Doesn't look like! because in that PR he is not upgrading serving.. so I will reopen this PR with his changes + updating the serving release.

File: pkg/reconciler/ingress/controller.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	reconciler_knative "knative.dev/pkg/reconciler"
```

File: pkg/reconciler/ingress/controller.go [hunk 5]
```diff
@@ -18,6 +18,7 @@ package ingress
 
 import (
 	"context"
+	knativeReconciler "knative.dev/pkg/reconciler"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go [hunk 6]
```diff
@@ -18,6 +18,7 @@ package ingress
...
 	"strings"
 	"time"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	knativeReconciler "knative.dev/pkg/reconciler"
```

**Comment by jmprusi** [comment]
waiting for https://github.com/knative/serving/pull/7470 to be merged, it fixes the integration tests.

File: pkg/reconciler/ingress/controller.go (hunk 5 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:21]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 6 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:23]: Format Go code:
```suggestion
	"time"

	knativeReconciler "knative.dev/pkg/reconciler"
```

**Comment by jmprusi** [comment]
retrying: `received 37.300000%, wanted 32.000000 +/- 5.000000`
/test pull-knative-net-kourier-integration-tests

**Comment by jmprusi** [comment]
/test pull-knative-net-kourier-integration-tests

several 404, some ingresses not getting ready.

**Pull request merged**